grin/src/AbstractInterpretation/HeapPointsTo/Result.hs

```diff
@@ -24,8 +24,8 @@ import AbstractInterpretation.HeapPointsTo.CodeGenBase (HPTMapping)
 type Loc = Int
 
 data SimpleType
-  = T_Int64
-  | T_Word64
+  = T_Int Word32
+  | T_Word Word32
```

Comment:
I would use Word32 as a parameter to define the concrete width of these types. Some ADT which is better aligned with the LLVM types would be better.

Reply:
I originally used `Word` but `llvm-hs-pure` uses `Word32`

Reply:
Ah got it, if LLVM-hs uses in the API than no objections from my side anymore :)